### PR TITLE
JWWEB-858 Added add contact page

### PR DESCRIPTION
**Pull request by Creepypoke**

### CODE REVIEW

File: src/pages/ContactsItemAdd/ContactsItemAdd.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  state: AppState,
+  {
+    address,
+    name,
+    note,
+  }: OwnProps,
+) {
+  const handledAddress = address || ''
+  const handledNote = note || ''
+  const handledName = name || 'Sara Jessica Parcel'
```
Comment (flexxnn) [inline src/pages/ContactsItemAdd/ContactsItemAdd.js]: ????
'Sara Jessica Parcel'

:)

Comment (Creepypoke) [reply src/pages/ContactsItemAdd/ContactsItemAdd.js]: I prepare the project to best future :D

File: src/pages/ContactsItemAdd/ContactsItemAddView.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+              label={t`Name`}
+              name='name'
+              isDisabled={submitting}
+            />
+            <Field
+              component={JInputField}
+              label={t`Address`}
+              name='address'
+              errorMessage={errors.address}
+              isDisabled={submitting || address}
```
Comment (IViol) [inline src/pages/ContactsItemAdd/ContactsItemAddView.js]: isDisabled is true when address exist and not empty. Is it correct?

Comment (Creepypoke) [reply src/pages/ContactsItemAdd/ContactsItemAddView.js]: Yeah, because address is not editable id in this case.

**Pull request merged**